README.md

```diff
@@ -27,10 +27,17 @@ First, add LeafErrorMiddleware as a dependency in your `Package.swift` file:
 ```swift
 dependencies: [
     // ...,
-    .package(name: "LeafErrorMiddleware", url: "https://github.com/brokenhandsio/leaf-error-middleware.git", from: "3.0.0")
+    .package(url: "https://github.com/brokenhandsio/leaf-error-middleware.git", from: "4.0.0")
 ],
 targets: [
-    .target(name: "App", dependencies: ["Vapor", ..., "LeafErrorMiddleware"]),
+    .target(
+        name: "App", 
+        dependencies: [
+            .product(name: "Vapor", package: "vapor"),
+            ..., 
+            .product(name: "LeafErrorMiddleware", package: "leaf-error-middleware")
```

Comment:
Name package name is `LeafErrorMiddleware` as described above - so this line should just be `"LeafErrorMiddleware"` as before